safe/gui/tools/minimum_needs/needs_profile.py

```diff
@@ -282,21 +283,17 @@ def provenance(self):
 
     @property
     def root_directory(self):
-        """Get the home root directory
+        """Map the root directory to ~/.qgis2/inasafe so the minimum needs
+           profile will be placed there (~/.qgis2/inasafe/minimum_needs).
 
         :returns: root directory
         :rtype: QString
         """
         if self._root_directory is None or self._root_directory == '':
```

Comment:
yes it is, but I don't know, I prefer to put it in the group of code which has similar functionality haha.